go-apps/meep-rnis/server/rnis.go

```diff
@@ -75,12 +76,16 @@ const MEAS_REP_UE_NOTIFICATION = "MeasRepUeNotification"
 const NR_MEAS_REP_UE_NOTIFICATION = "NrMeasRepUeNotification"
 
 var RNIS_DB = 5
+var DATA_DB = 0
+var RNIS_DB_CONNECTOR_INDEX = 0
+var DATA_DB_CONNECTOR_INDEX = 1
 
-var rc *redis.Connector
+var rc [2]*redis.Connector
```

Comment:
Instead of using a redis connector directly, use the meep-metric-store package to access the network metrics. You can then revert all the changes make to create a slice of connectors.

Reply:
fixed